python/pyspark/sql/group.py

```diff
@@ -214,11 +214,11 @@ def apply(self, udf):
 
         :param udf: A function object returned by :meth:`pyspark.sql.functions.pandas_udf`
 
-        >>> from pyspark.sql.functions import pandas_udf
+        >>> from pyspark.sql.functions import pandas_udf, PandasUdfType
         >>> df = spark.createDataFrame(
         ...     [(1, 1.0), (1, 2.0), (2, 3.0), (2, 5.0), (2, 10.0)],
         ...     ("id", "v"))
-        >>> @pandas_udf(returnType=df.schema)
+        >>> @pandas_udf(returnType=df.schema, functionType=PandasUdfType.GROUP_FLATMAP)
```

Comment:
I think `GROUP_MAP` is better here, think about `RDD.mapPartitions`, we pass a function that takes an `Iterator`(group) and returns another `Iterator`(group). `GROUP_TRANSFORM` is also fine.

Reply:
I am not tied to the current naming. Let me think a bit more on this.